cfgov/v1/models/sublanding_page.py

```diff
@@ -59,3 +62,12 @@ class SublandingPage(CFGOVPage):
     ])
 
     template = 'sublanding-page/index.html'
+
+    def get_browsefilterable_posts(self, request):
+        filter_pages = [p.specific for p in self.get_descendants()
+                        if 'BrowseFilterablePage' in p.specific_class.__name__]
+        posts = []
+        for page in filter_pages:
+            posts.append(page.get_page_set(page.get_form_class()(parent=page),
```

Comment:
Yes. I could move it to a more readable format though. 

`page.get_form_class()` returns the form class that's callable. `parent=page` must be passed for the init to create the form properly.